datafusion/optimizer/src/decorrelate_where_exists.rs

```diff
@@ -20,7 +20,7 @@ use crate::utils::{
     verify_not_disjunction,
 };
 use crate::{utils, OptimizerConfig, OptimizerRule};
-use datafusion_common::{context, plan_err, DataFusionError};
+use datafusion_common::context;
```

Comment:
keep `DataFusionError`, following code we can use `Result` instead of `datafusion_common::Result`

Reply:
Or maybe we could just do `use datafusion_common::Result`